synapse/rest/client/v1/login.py

```diff
@@ -401,6 +423,52 @@ async def _do_jwt_login(self, login_submission: JsonDict) -> Dict[str, str]:
         return result
 
 
+class RestoreDeviceServlet(RestServlet):
+    PATTERNS = client_patterns("/org.matrix.msc2697/restore_device")
```

Comment:
Not sure if this is supposed to show up only in `unstable` or also `r0` (the current code would do both).